R/tm_a_mmrm.R

```diff
@@ -1030,8 +1030,15 @@ srv_mmrm <- function(id,
       anl_name = "ANL_ADSL"
     )
 
+    # Set tern default for missing values for reproducibility (on .onLoad for the examples)
+    add_options <- reactive({
+      within(data(), {
+        tern::set_default_na_str("<Missing>")
+      })
```

Comment:
2 questions:

- Should we get the default from the app's R options?
- Is there a better name for the variable? `add_options` sounds like a function, I've proposed a verbose one, but if you like the current one, then place `_r` suffix.

```suggestion
    data_with_tern_options_r <- reactive({
      within(data(), {
        tern::set_default_na_str(default_na_str)
      }, default_na_str = getOption("tern_default_na_str", default = "<Missing>"))
```

Reply:
same would apply to other instances of this code snippet